src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -72,6 +73,42 @@ export const selectTotalStakingDelegationCryptoByAccountSpecifier = createSelect
   }
 )
 
+export const selectTotalStakingDelegationCrypto = createSelector(
+  selectStakingData,
+  // We make the assumption that all delegation rewards come from a single denom (asset)
+  // In the future there may be chains that support rewards in multiple denoms and this will need to be parsed differently
+  stakingData => {
+    let total = bnOrZero(0)
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for (const [_, value] of Object.entries(stakingData.byAccountSpecifier)) {
+      const amount = reduce(
+        value?.delegations,
+        (acc, delegation) => acc.plus(bnOrZero(delegation.amount)),
+        bn(0)
+      )
+      total = total.plus(amount)
+    }
+
+    return total.toString()
+  }
+)
+
+export const selectTotalStakingDelegationFiat = createSelector(
+  selectTotalStakingDelegationCrypto,
+  selectMarketData,
+  // We make the assumption that all delegation rewards come from a single denom (asset)
+  // In the future there may be chains that support rewards in multiple denoms and this will need to be parsed differently
+  (totalStaked, md) => {
+    //TODO this wont work when we support more than only cosmos staking
+    const cosmosPrice = md['cosmos:cosmoshub-4/slip44:118']?.price
```

Comment:
if we change the selector above to return an object keyed by asset id rather than a scalar, we don't have to hard code this here.